tests/dummy/app/components/task-function-syntax-5/component.js

```diff
@@ -1,30 +1,24 @@
 // BEGIN-SNIPPET task-function-syntax-5
-import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
+import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { task } from 'ember-concurrency';
 
-export default class MyOctaneComponent extends Component {
-  @tracked status = null
+export default Component.extend({
```

Comment:
What's happening here? Seems like a revert to old school Ember?

Reply:
Yeah, instead of documenting Octane use (which we do throughout), I wanted to maintain a documented use with Classic Ember.